crates/sui-core/src/authority_active/execution_driver/mod.rs

```diff
@@ -101,9 +101,10 @@ where
     // Before executing de-duplicate the list of pending trasnactions
     let mut seen = HashSet::new();
     let mut indexes_to_delete = Vec::new();
-    let pending_transactions: Vec<_> = pending_transactions
+
+    let (mut pending_sequenced, pending_transactions): (Vec<_>, Vec<_>) = pending_transactions
         .into_iter()
-        .filter(|(idx, digest)| {
+        .filter(|(idx, (_, digest))| {
```

Comment:
I think here we need to favor digests that are sequenced, if there is duplicate.

Reply:
correct, we first serially execute all sequenced digests.